lib/smart_proxy_dynflow_core/core.rb

```diff
@@ -73,6 +76,14 @@ def ensure_initialized
         @instance
       end
 
+      def plugin_matchers
```

Comment:
From the name it's not clear, what kind of matchers it is. What about `silencer_matchers`?